kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/DeploymentOperationsImpl.java

```diff
@@ -137,9 +140,11 @@ public void run() {
     ScheduledFuture poller = executor.scheduleWithFixedDelay(deploymentPoller, 0, POLL_INTERVAL_MS, TimeUnit.MILLISECONDS);
     try {
       if (Utils.waitUntilReady(queue, getConfig().getScaleTimeout(), TimeUnit.MILLISECONDS)) {
-        int currentReplicas = atomicDeployment.get().getStatus().getReplicas() != null ? atomicDeployment.get().getStatus().getReplicas() : 0;
-        LOG.error("Only {}/{} pod(s) ready for Deployment: {} in namespace: {} - giving up",
-          currentReplicas, atomicDeployment.get().getSpec().getReplicas(), atomicDeployment.get().getMetadata().getName(), namespace);
+        LOG.debug("{}/{} pod(s) ready for Deployment: {} in namespace: {}.",
+          replicasRef.get(), count, name, namespace);
+      } else {
+        LOG.error("{}/{} pod(s) ready for Deployment: {} in namespace: {}  after waiting for {} seconds so giving up",
+          replicasRef.get(), count, name, namespace, getConfig().getScaleTimeout(), TimeUnit.MILLISECONDS);
```

Comment:
Wrong logging here for TimeUnit.MILLISECONDS - assume you're trying to convert the scale timeout in millis to seconds? `TimeUnit.MILLISECONDS.toSeconds(getConfig().getScaleTimeout())` perhaps?